src/NJsonSchema.SourceGenerators.CSharp/NJsonSchema.SourceGenerators.CSharp.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="NJsonSchema.SourceGenerators.CSharp.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <LangVersion>latest</LangVersion>
+    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IsRoslynComponent>true</IsRoslynComponent>
+    <NoWarn>1701;1702;NU5128</NoWarn>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <ProjectReference Include="..\NJsonSchema.CodeGeneration.CSharp\NJsonSchema.CodeGeneration.CSharp.csproj" PrivateAssets="All">
```

Comment:
With `PrivateAssets="All"` it will not add `NJsonSchema.CodeGeneration.CSharp` as a dependency of `NJsonSchema.SourceGenerators.CSharp` project as I assume it's not needed. Code from `NJsonSchema.SourceGenerators.CSharp` should be enough to generate C# classes. For more advanced scenarios `NJsonSchema.CodeGeneration.CSharp` can be installed explicitly in client's projects.